bouncer/blacklist/brain.py

```diff
@@ -0,0 +1,26 @@
+from blacklist import models
+
+
+def is_ip_blacklisted(ip):
+    """
+    Checks if ip is in IPEntry model.
+    """
+
+    if models.IPEntry.objects.filter(entry_value=ip):
+        return True
+    return False
+
+
+def is_email_blacklisted(email):
+    """
+    Checks if email is in EmailEntry model host is in HostEntry model.
+    """
+
+    if models.EmailEntry.objects.filter(entry_value=email):
+        return True
+    else:
```

Comment:
`else` is not really needed here, if the condition is True, the function will return. It's always easier to read code with early returns, as it has less nests.